uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave-calendar.scss

```diff
@@ -68,8 +53,23 @@ td.chr_leave-calendar__name {
   }
 }
 
-//Public Holiday, Weekend and Non Working day needs to be specified before other styles
-//to give other styles more priority
+.chr_leave-calendar__item--disabled {
```

Comment:
I think we have `.chr_disabled{opacity:0.5;pointer-events:none}` - will that work? Or do you really need cursor: not-allowed; for UX?